### PR TITLE
Add support for alternate links

### PR DESCRIPTION
**Pull request by vic0707**

### CODE REVIEW

File: parser/sitemapIndexStreamParser.js [hunk 1]
```diff
@@ -27,8 +28,8 @@ function sitemapIndexStreamParser (openingTag, maxUrls) {
           break;
         case 3:
           if (validSitemap(openingTag, currentSitemap)) {
-            chlidCounter[`${tag}`] ? chlidCounter[`${tag}`]++ : chlidCounter[`${tag}`] = 1;
-            if (chlidCounter[`${tag}`] > 1) {
+            childCounter[`${tag}`] ? childCounter[`${tag}`]++ : childCounter[`${tag}`] = 1;
```
Comment (Mickael-van-der-Beek) [inline parser/sitemapIndexStreamParser.js]: No need for template tags here: `childCounter[`${tag}`] === childCounter[tag]`

Comment (mklinga) [inline parser/sitemapIndexStreamParser.js]: Seems to be earlier, but maybe could clean this sentence a bit - it seems quite nasty to me to do assignments inside ternary...

File: parser/urlSetStreamParser.js [hunk 2]
```diff
@@ -27,9 +28,17 @@ function urlSetStreamParser (openingTag, domain, maxUrls) {
           break;
         case 3:
           if (validSitemap(openingTag, currentSitemap)) {
-            chlidCounter[`${tag}`] ? chlidCounter[`${tag}`]++ : chlidCounter[`${tag}`] = 1;
-            if (chlidCounter[`${tag}`] > 1) {
+            childCounter[`${tag}`] ? childCounter[`${tag}`]++ : childCounter[`${tag}`] = 1;
```
Comment (mklinga) [inline parser/urlSetStreamParser.js]: Same thing here

File: parser/sitemapIndexStreamParser.js [hunk 3]
```diff
@@ -27,8 +28,8 @@ function sitemapIndexStreamParser (openingTag, maxUrls) {
           break;
         case 3:
           if (validSitemap(openingTag, currentSitemap)) {
-            chlidCounter[`${tag}`] ? chlidCounter[`${tag}`]++ : chlidCounter[`${tag}`] = 1;
-            if (chlidCounter[`${tag}`] > 1) {
+            childCounter[tag] = (childCounter[tag] || 0) + 1;
```
[the remark this answers is not in the document]
Comment (vic0707) [reply parser/sitemapIndexStreamParser.js]: Credit to @mklinga

**Pull request merged**